src/app.ts

```diff
@@ -535,6 +642,26 @@ function registerPublicRoutes(app: Express): void {
     res.status(200).json(serverInfo);
   });
 
+  app.post("/spidMsgPost", (req, res) => {
+    const { QueueClient } = require("@azure/storage-queue");
+    const spidQueueClient = new QueueClient(
+      queueConnectionString,
+      spidQueueName
+    );
+
+    spidQueueClient.create();
+
+    const sampleMsg = {
+      ...req.body
+    };
+
+    const buffer1 = Buffer.from(JSON.stringify(sampleMsg));
+    const sendMessageResponse = spidQueueClient.sendMessage(
+      buffer1.toString("base64")
+    );
+    res.status(200).json(sendMessageResponse.requestId);
+  });
```

Comment:
what's tihs endpoint used for ?

Reply:
It was fir testing purposes. I'll delete it